### PR TITLE
Make the bdgg emote formatter a proper formatter.

### PR DESCRIPTION
**Pull request by xDashh**
This hopefully doesn't break too much and enables halloween memes again.

Removed some moobie flairs so they might mean something again in the future.

### CODE REVIEW

**Comment by PinkishCow** [comment]
The stuff I did was all based around utc and gave a wide range for the times it was active so that should cover Halloween everywhere when you get around to it 👌

**Review by PinkishCow — APPROVED** [review]
I found this button and felt like clicking it to see what it did

File: betterdgg/modules/spooky.js [hunk 1]
```diff
@@ -8,12 +8,14 @@
...
-        "WhoahDude": [ 'blink' ]
+        "WhoahDude": [ 'blink' ],
+        "YEE": [ 'crawl' ],
+        "NOBULLY": [ 'blink' ] //TODO change effect
     };
     var EMOTE_RE = new RegExp("\\b(?:bdgg-)?chat-emote-(" + Object.keys(EMOTES).join('|') + ")");
 
     bdgg.spooky = (function() {
-        var END = moment('2014-11-01 05:00');
+        var END = moment('2016-11-01 05:00'); //TODO verify time and possibly take care of timezones
```
Comment (kenany) [inline betterdgg/modules/spooky.js:18]: If we want this to end at the same time for everyone, `moment.utc` might be a sufficient solution.

**Review by kenany — APPROVED** [review]

File: betterdgg/modules/spooky.js (hunk 1 above)
Comment (PinkishCow) [reply betterdgg/modules/spooky.js:18]: Its already done here, https://github.com/PurpleCox/betterdgg/blob/highcolour/betterdgg/modules/spooky.js 
 just other parts did not work so I told @xDashh and @BryceMatthes to fix that first 👌

Comment (downthecrop) [reply betterdgg/modules/spooky.js:18]: I'll pasta that in 👍

**Pull request merged**